client/packages/requisitions/src/ResponseRequisition/DetailView/ResponseLineEdit/ResponseLineEdit.tsx

```diff
@@ -26,6 +29,29 @@ export const ResponseLineEdit = ({
   const { draft, isLoading, save, update } =
     useDraftRequisitionLine(currentLine);
   const { next, hasNext } = useNextResponseLine(currentLine);
+  const { data } = useResponse.line.stats(draft?.id);
+
+  const tabs = [
+    {
+      Component: <ResponseStoreStats
+        stockOnHand={data?.responseStoreStats.stockOnHand || 0}
+        incomingStock={data?.responseStoreStats.incomingStock || 0}
+        stockOnOrder={data?.responseStoreStats.stockOnOrder || 0}
+        requestedQuantity={data?.responseStoreStats.requestedQuantity || 0}
+        otherRequestedQuantity={data?.responseStoreStats.otherRequestedQuantity || 0}
+      />,
+      value: 'My Store',
```

Comment:
can these be translations please?

Reply:
Should the other tab values also be in translations as well then? (Details, Log)